api/src/services/db.ts

```diff
@@ -1,14 +1,23 @@
 import knex from 'knex';
+import { DateTime, IANAZone } from 'luxon';
 
 import { findConfig } from './configService';
 
+// Calculate the numerical time zone offset for the specified time zone
+// at the current moment. (mysqljs doesn't support IANA time zones.)
```

Comment:
Is this a comment that still needs to be on the file?

Reply:
Thank you for your feedback; I was uncertain about this because the code is non-obvious. I've removed the comment.